docs/ssm-deployment-guide.md

```diff
@@ -229,18 +231,18 @@ Enter into ${SMART_HOME} directory for running SSM. You can type `./bin/ssm vers
 ----------------------------------------------------------------------------------
```

Comment:
Line 201,
If you meet any problem, please open the smartserver-$hostname-$user.log under ${SMART_HOME}/logs directory. All the trouble shooting clues are there.
->
If you meet any problem, please try to find trouble shooting clues in log file smartserver-$hostname-$user.log under ${SMART_HOME}/logs directory.